test/functional/sqlsrv/TC43_FetchData.phpt

```diff
@@ -45,11 +44,13 @@ function FetchFields()
                 fatalError("Field $j of Row $i is missing");
             }
             $col = $j + 1;
-            if (!IsUpdatable($col)) {
-                $skipCount++;
-            } else { // should check data even if $fld is null
-                $data = getInsertData($startRow + $i, $col, $skipCount);
-                if (!CheckData($col, $fld, $data)) {
+            $data = AE\getInsertData($startRow + $i, $col);
```

Comment:
how come you getInsertData before checking if $col isUpdatable?

Can you also explain what $skipCount does and why you got rid of it?

Reply:
Only the timestamp column (column 27) is updatable, and the original way to look for insert data is to skip this column. I'll explain more if this is unclear.

Reply:
OK I'm beginning to understand what $skipCount is for now. But I still don't understand why you have getInsertData in line 47. There is an identical line in line 50, and $data is only used inside the isUpdatable if block.

Reply:
Yes, it was an oversight. Thanks!